.github/workflows/ci.yml

```diff
@@ -134,13 +124,12 @@ jobs:
       AWS_SECRET_ACCESS_KEY: fake-aws-key
       TARGET: x86_64-unknown-linux-gnu
       USE_CONTAINER: none
-      FEATURES: cloudwatch-logs-integration-tests,cloudwatch-metrics-integration-tests,ec2-metadata-integration-tests,firehose-integration-tests,kinesis-integration-tests,s3-integration-tests
     steps:
       - uses: actions/checkout@v1
-      - run: docker-compose up -d localstack mockwatchlogs ec2_metadata minio
-      - run: cargo test --no-default-features --features $FEATURES
+      - run: make slim-builds
+      - run: make test-integration-docker
 
-  docker-integration-tests:
+  test-integration-elasticsearch:
```

Comment:
I do kind of think keeping the templating from #2383 was useful here and for future work.